srcpkgs/lilypond/template

```diff
@@ -1,28 +1,27 @@
 # Template file for 'lilypond'
 pkgname=lilypond
```

Comment:
we should add a notice near the top of lilypond and lilypond-doc that they should be kept in sync like it's done in vala and valadoc